Jenkinsfile

```diff
@@ -171,6 +171,19 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             }
         }
 
+        stage('QEMU Kola Run') {
+            utils.shwrap("""
+            coreos-assembler kola run
+            tar -cf - tmp/kola/ | xz -c9 > _kola_temp.tar.xz
+            """)
+            archiveArtifacts: "_kola.tar.xz"
+            // archive the image if the tests failed
+            def kolaResult = utils.shwrap_capture("jq -r '.result' < tmp/kola/reports/report.json")
+            if (kolaResult == "FAIL") {
+                archiveArtifacts: "builds/${newBuildID}/fedora-coreos-${newBuildID}-qemu.qcow2"
```

Comment:
I think we also want to return early in this case? E.g. something like:

```
currentBuild.result = 'FAILURE'
return
```

Though one gotcha IIRC is that the `return` can't be in a `stage`; it has to be at the "top-level" script.